src/libraries/Common/src/Internal/Cryptography/SymmetricPadding.cs

```diff
@@ -0,0 +1,220 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
```

Comment:
Same general "don't add to this bad namespace, please" (unless it's just splitting a class into partials)